### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by stone100010** [facebookresearch/pifuhd#27]
Is it convenient to tell me your training data set? Train the data set of "pifuhd.pt" file.Thanks

**Comment by shunsukesaito** [comment]
Please find the full list of our dataset in the link below. By subtracting the test list, you can use the remaining for training and validation. Our released model is trained with all subtracting the test set.
https://github.com/facebookresearch/pifuhd/tree/master/data
